Web/starter_project_iconics/store/features/api/blog-api.ts

```diff
@@ -0,0 +1,19 @@
+import { Blog } from '@/types/blog/blog'
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
+
+const API_BASE_URL = 'http://localhost:3500'
+
+export const blogApi = createApi({
+  reducerPath: 'api/',
+  baseQuery: fetchBaseQuery({ baseUrl: API_BASE_URL }),
+  endpoints: (builder) => ({
+    getBlogs: builder.query<Blog[], void>({
+      query: () => 'blogs',
```

Comment:
change the reducer path to make it more descriptive, in this case the reducerPath should be something like "blog"